File: get_console_errors.py
import asyncio
from playwright.async_api import async_playwright

async def main():
    async with async_playwright() as p:
        browser = await p.chromium.launch()
        page = await browser.new_page()
        
        # Capture console messages
        page.on("console", lambda msg: print(f"Console {msg.type}: {msg.text}"))
        page.on("pageerror", lambda err: print(f"Page Error: {err}"))
        
        await page.goto("http://127.0.0.1:5000", wait_until="networkidle")
        
        try:
            await page.click("#openBidBtn")
            print("Clicked #openBidBtn successfully.")
            await page.wait_for_timeout(1000)
        except Exception as e:
            print("Click error:", e)
            
        await browser.close()

asyncio.run(main())

File: get_console_errors.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.new_page();
  
  page.on('console', msg => {
    if (msg.type() === 'error') {
      console.log(`Console Error: ${msg.text()}`);
    }
  });

  page.on('pageerror', error => {
    console.log(`Page Error: ${error.message}`);
  });

  await page.goto('http://127.0.0.1:5000', { waitUntil: 'networkidle' });
  
  // Try to click the bid button
  await page.click('#openBidBtn').catch(e => console.log("Click error:", e.message));
  
  await browser.close();
})();
